js/parts/PieSeries.js

```diff
@@ -597,15 +597,22 @@ seriesType('pie', 'line',
                     // start values
                     graphic.attr({
                         // animate from inner radius (#779)
-                        r: point.startR || (series.center[3] / 2),
+                        // If chart is inverted polar and series is column,
+                        // allow 0 as radius
+                        r: series.isRadialBar ?
+                            (args).start === (args).end ?
```

Comment:
Do we need brackets in args?

Reply:
It is already corrected in the newest version.